src/interfaces/TodoProps.ts

```diff
@@ -0,0 +1,7 @@
+import { TodoData } from "./TodoData";
+
+export interface TodoProps {
+  todoArr: TodoData[];
+  editTodo(taskID: number): void;
```

Comment:
naming is inconsistent, either use `editTodo` and `deleteTodo` or `handleDelete` and `handleEdit`. I will prefer the first pair as it gives more context.

Reply:
understood